src/routes/auth.route.js

```diff
@@ -26,13 +29,35 @@ router.route('/update-password')
           authCtrl.updatePassword);
 
 router.route('/reset-password')
+// Setting `AUTH_SERVICE_REQUIRE_ACCOUNT_VERIFICATION` or
+// `AUTH_SERVICE_REQUIRE_SECURE_ACCOUNT_VERIFICATION` to `true` will force a
+// user to verify their email address before this endpoint will 200.
     .post(validate(authValidation.resetToken),
           authCtrl.resetToken);
 
 router.route('/reset-password/:token')
+// This endpoint specifically does not require email verification.
     .post(validate(authValidation.resetPassword), checkPassword,
           authCtrl.resetPassword);
 
+router.route('/dispatch-verification-request')
+// Triggers the dispatching of a verification request for a messaging protocol.
+// Currently only supports sending a verification email for the email address
+// provided in the User's email column.
+    .post(validate(authValidation.dispatchVerificaitonRequest),
+          authCtrl.dispatchVerificaitonRequest);
+
+router.route('/provide-verifying-user')
```

Comment:
Can you rename this to `/get-verifying-user` and change everything in `orange-web` accordingly?

Related: It would be nice if it was an HTTP `GET` to `/verifying-user?token=THE_TOKEN`, but I recognize that we probably want the token in the request body, as it is, for security purposes.